packages/materialized/src/fields/material-date.field.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import {
+  and,
+  FieldProps,
+  formatIs,
+  handleChange,
+  mapStateToInputProps,
+  RankedTester,
+  rankWith,
+  registerStartupInput,
+  uiTypeIs
+} from 'jsonforms-core';
+import { connect } from 'react-redux';
+
+import Input from 'material-ui/Input';
+
+export const MaterialDateField = (props: FieldProps) => {
```

Comment:
An empty date overlaps with its label initially.

Reply:
I think this is a bug in material ui